lib/msal-angular/docs/v2-docs/events.md

```diff
@@ -41,8 +41,50 @@ export class AppComponent implements OnInit, OnDestroy {
 ```
 
 
-For the full example of using events, please see the sample [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/samples/msal-angular-v2-samples/angular10-sample-app/src/app/app.component.ts#L29).
+For the full example of using events, please see our sample [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/samples/msal-angular-v2-samples/angular10-sample-app/src/app/app.component.ts#L29).
 
 ## Table of events
 
 For more information about the `EventMessage` object, including the full table of events currently emitted by `@azure/msal-browser` (including descriptions and related payloads), please see the documentation [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/events.md).
+
+## Handling errors with events
+
+As the `EventError` in `EventMessage` is defined as `AuthError | Error | null`, an error should be validated as the correct type before accessing specific properties on it. 
```

Comment:
I wonder if more should be said on the browser docs about this too? Would this be useful for msal-react? @jasonnutter @tnorling

Reply:
Yeah probably not a bad idea.

Reply:
Yes, I think a note in the events doc in browser would be useful. I'm going to put together an error handling doc for react soon so don't worry about that for now.

Reply:
Makes sense. Browser doc updated.